ext/PlotlyJSExt.jl

```diff
@@ -1,8 +1,12 @@
+module PlotlyJSExt
+
 import JSON
+import PlotlyJS
 
 function DashBase.to_dash(p::PlotlyJS.SyncPlot)
```

Comment:
Strictly speaking, shouldn't this go in `DashBase.jl`?

Reply:
Yes, good catch!